source/lib/rocprof-sys/library/ompt.cpp

```diff
@@ -128,14 +125,13 @@ struct ompt : comp::base<ompt, void>
         if(_cid > 0)
         {
             category_region<category::ompt>::stop<tim::quirk::perfetto>(
-                (_ctx_info.func.empty()) ? m_prefix : _ctx_info.func, _ts,
+                _ctx_info.label, _ts, ::perfetto::Flow::ProcessScoped(_cid),
```

Comment:
```suggestion
                _ctx_info.label, _ts,
```
This change is creating a circular flow in the ompt event. See image below. 

![image](https://github.com/user-attachments/assets/532c330c-33c7-4067-b2f1-9cc66b6d38cb)

Let's just remove the condition, since both blocks are doing the same thing.